crex/models/course.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+from typing import Tuple, FrozenSet
+from crex.models import Department, CourseCode, TopicArea
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+from frex.models import DomainObject
+from rdflib import URIRef
+
+
+@dataclass_json
+@dataclass(frozen=True)
+class Course(DomainObject):
+    name: str
```

Comment:
course content may change later based on changes to YACS' course catalog scraping